crates/core/src/subscription/query.rs

```diff
@@ -404,46 +392,6 @@ mod tests {
         Ok(())
     }
 
-    #[test]
```

Comment:
Maybe we want to check that the row values are equal?

Reply:
Yes, but technically we already have tests for that.